reference/configuration/doctrine.rst

```diff
@@ -398,7 +398,7 @@ The following block shows all possible configuration keys:
                 # the DBAL driverOptions option
                 options:
                     foo: bar
-                path:                 '%kernel.root_dir%/data/data.sqlite'
+                path:                 '%kernel.project_dir%ta/data.sqlite'
```

Comment:
The original was `'%kernel.root_dir%/data/data.sqlite'`, so the new one should be `'%kernel.project_dir%/app/data/data.sqlite'`